statsprinter.go

```diff
@@ -113,6 +113,18 @@ func (p *planePrinter) printStatistics(s stats) {
 		colorYellow("retried to resolve hostname ")
 		colorRed("%d ", s.retriedHostnameResolves)
 		colorYellow("times\n")
+
+		if len(s.hostnameChanges) >= 2 {
```

Comment:
hmmm, why not `>= 1`?

Imagine if there were only one IP address change throught the lifetime of the program.

Reply:
So, `hostnameChanges` stores the changes a slice of [toAddr, when] pair/values. In order to represent a single change, we need a slice with a len of at least 2, where `hostnameChange[i]`=from, and `hostnameChange[i+1]`=to.

The slice is always initialized using the first resolved ip as a starting point inside `processUserInput()`
https://github.com/pouriyajamshidi/tcping/blob/b13f0443326d130c912cab50032f38ca2abcb7b6/tcping.go#L262-L269

Reply:
yea yea, I was too sleepy while reviewing 😄